core/email_api.php

```diff
@@ -1635,6 +1640,12 @@ function email_build_visible_bug_data( $p_user_id, $p_bug_id, $p_message_id ) {
 	$t_category_name = category_full_name( $t_row['category_id'], false );
 	$t_bug_data['email_category'] = $t_category_name;
 
+	if( count( $t_tag_rows = tag_bug_get_attached( $p_bug_id ) ) !== 0 && access_compare_level( $t_user_access_level, config_get( 'tag_view_threshold' ) ) ) {
+		foreach( $t_tag_rows as $t_tag ) {
```

Comment:
I would use implode() with ', ' (comma + space) as the separator instead of the for loop.